### PR TITLE
Added HHVM fatals to error handler

### PR DESCRIPTION
**Pull request by samdark**
#8933

### CODE REVIEW

File: framework/base/ErrorException.php [hunk 1]
```diff
@@ -17,6 +17,8 @@
  */
 class ErrorException extends \ErrorException
 {
+    const E_HHVM_FATAL_ERROR = 16777217;
```
Comment (klimov-paul) [inline framework/base/ErrorException.php]: Is not such constant defined at HHVM environment already?

Comment (klimov-paul) [inline framework/base/ErrorException.php]: Also, is this the only one extra error added at HHVM?

Comment (cebe) [inline framework/base/ErrorException.php]: where is this constant from, it does not seem to be documented: http://docs.hhvm.com/manual/en/errorfunc.constants.php

Comment (dieend) [inline framework/base/ErrorException.php]: 16777217 comes from here: https://github.com/facebook/hhvm/blob/master/hphp/runtime/base/runtime-error.h#L62 More info is in this issue https://github.com/facebook/hhvm/issues/4818 in comment from iansltx.

**Comment by petrabarus** [comment]
As I reported in #8960, this fix does show the error. But it doesn't show the stack trace.

**Pull request closed without merging**